pkg/apis/garden/validation/validation.go

```diff
@@ -1697,6 +1697,12 @@ func validateKubeControllerManager(kubernetesVersion string, kcm *garden.KubeCon
 		allErrs = append(allErrs, field.Invalid(fldPath, kubernetesVersion, err.Error()))
 	}
 	if kcm != nil {
+		if maskSize := kcm.NodeCIDRMaskSize; maskSize != nil {
+			fldPath = fldPath.Child("nodeCIDRMaskSize")
```

Comment:
You don't need this here since you already use `fldPath.Child("nodeCIDRMaskSize")` in line 1703.